nextflow.config

```diff
@@ -106,7 +106,7 @@ profiles {
        }
       }
     } else if (params.illumina) {
-        process.container = "docker://genomicmedicinesweden/gms-artic-illumina:latest"
+        process.container = "docker://henningonsbring/artic-ncov2019-illumina:latest"
```

Comment:
Do you want to change the container?

Reply:
I want to change back to `process.container = "docker://genomicmedicinesweden/gms-artic-illumina:latest"` when there is a new container on gms dockerhub that works. This was a temporar fix to see if it works.